recipes/serializers.py

```diff
@@ -0,0 +1,45 @@
+from .models import Recipe, Ingredient
+from rest_framework import serializers
+
+
+class ChoiceField(serializers.ChoiceField):
+    def to_representation(self, obj):
+        return self._choices[obj]
+
+
+class IngredientSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = ('name',)
+        extra_kwargs = {
```

Comment:
why did you remove the validator of name field?

Reply:
If I don't it checks whether the ingredient already exists or not and if so it raises a validation error, which prevent me adding new recipes with existing ingredients. I don't want this validation check since in create method I manually check if an ingredient exists or not. Some validators still works though (e.g. allow_blank).